### PR TITLE
Updates Submodules to latest 

### PR DESCRIPTION
**Pull request by Floix**
Updates Submodules to latest instead of latest at the time of the last main repository commit.

Added installation of pip requirements during installation.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,8 +35,8 @@ clean:
 
 install:
 	git submodule init
-	git submodule update
-	(cd ${TOOLSDIR}/; python3 setup.py install --install-scripts=${DESTDIR}/bin)
+	git submodule foreach git pull origin master
```
[the remark this answers is not in the document]
Comment (Floix) [reply Makefile:38]: I updated the make file accordingly

[the remark this answers is not in the document]
Comment (gunnarx) [reply Makefile:38]: I'm not convinced of this yet.  I continued the discussion in https://github.com/GENIVI/vss-tools/issues/10 where we have the context.

**Review by gunnarx — CHANGES_REQUESTED** [review]
Formal review added, see other comments for details

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (gunnarx) [reply Makefile:38]: > I updated the make file accordingly

@Floix I don't see that change, did you push?

**Comment by danielwilms** [comment]
After yesterdays discussion, I would use the reference to the tools, which is defined in the submodule and update that accordingly instead of always using simply the latest version. But the extension to run pip with the requirements file makes sense for me.

**Comment by gunnarx** [comment]
I would fix it but by convention I don't force push to update other people's PRs if they are sent from their master branch.

**Comment by danielwilms** [comment]
#184 will update the submodules, so I'll close this one.

**Pull request closed without merging**